examples/int8_training/Finetune_flan_t5_large_bnb_peft.ipynb

```diff
@@ -0,0 +1,8321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def preprocess_function(examples):
</pre><p>A space or two before the function would be nice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/peft/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>